From: Paolo Bacchilega <paobac@src.gnome.org>
Date: Tue, 11 Jan 2022 12:30:44 +0100
Subject: LINGUAS: removed duplicated entry

(cherry picked from commit daef485dffd67319c114cf1bda453bf6157e11ff)
---
 help/LINGUAS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/help/LINGUAS b/help/LINGUAS
index 0e8a52d..fa1fc3e 100644
--- a/help/LINGUAS
+++ b/help/LINGUAS
@@ -8,7 +8,6 @@ eu
 fr
 hu
 id
-id
 nl
 oc
 sl
